### PR TITLE
Fix tiny typo in bread types.

### PR DESCRIPTION
**Pull request by kdungs**
Moin Hendrik,

while reading through your amazing book, I stumbled over the tiniest of typos. Feel free to reject the PR and just fix it yourself. 😄 And keep up the great work!

Cheers,
Kevin

### CODE REVIEW

**Comment by cedounet** [comment]
Did you cross-check with the numerous corrections that were made to the physical book ? https://discord.com/channels/1159886299243749477/1163969309807235092/1194759730736922735

Thanks!

C/

**Comment by kdungs** [comment]
Apologies, I'm not on the Discord and naïvely assumed that the GitHub repo would be the single source of truth 🙈. Feel free to close this if it's redundant.

**Comment by cedounet** [comment]
I will merge that one in any case... but my point was (unfortunately?) there is another ground truth.  I reckon @hendricius said it was about 1000 changes that need to be merged back.

**Comment by cedounet** [comment]
@hendricius  build of Dicker image is broken 😢

**Comment by hendricius** [comment]
@cedounet is it broken in general, or just for this PR?

**Comment by hendricius** [comment]
Merging - will have a look at on the main branch

**Comment by cedounet** [comment]
The PR is very harmless… so expecting it is a bug with CI… interesting it built on main though but not on PR.

**Comment by hendricius** [comment]
@cedounet maybe a temporary glitch in the matrix. Fingers crossed.

**Comment by cedounet** [comment]
i was going to try it with the other one.. but unfortunately i pushed to main directly.  So let me try again.

BTW, you should maybe disable push to main directly. 9 times out of 10 it is a mistake to do so.

**Comment by hendricius** [comment]
@cedounet I'll leave push to main there for now - with great power comes great responsibility 😎
